### PR TITLE
docs: add zh docs

### PR DESCRIPTION
**Pull request by Mister-Hope**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

**Comment by Mister-Hope** [comment]
> Thanks a lot. Could you leave formatting to another PR?

Yes, and BTW, you mixed 120 wrap and no wrap in the English docs, how should I fomat them?

**Comment by Mister-Hope** [comment]
This PR only adds zh docs now

File: docs/zh/guide/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# 介绍
+
+[VuePress](https://v1.vuepress.vuejs.org/zh/) 是可扩展的静态网站生成器，使用 [Vue.js](https://github.com/vuejs/zh/), 其创建目的是为了满足 Vue 自己的子项目的文档需求。
```
Comment (newsbielt703) [inline docs/zh/guide/README.md:3]:
```suggestion
[VuePress](https://v1.vuepress.vuejs.org/zh/) 是由 Vue 驱动的可扩展静态网站生成器，其创建目的是为了满足 Vue 子项目的文档需求。
```

File: docs/zh/guide/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+后来，引入 [插件系统](https://v1.vuepress.vuejs.org/zh/plugin/), 大大改善了 VuePress 的可扩展性。该项目是由 VuePress 1.x 的作者创建的 VuePress 插件，其目的是减少使用开发 VuePress 博客主题子程序的成本。
```
Comment (newsbielt703) [inline docs/zh/guide/README.md:5]:
```suggestion
后来，引入 [插件系统](https://v1.vuepress.vuejs.org/zh/plugin/)的 VuePress 1.x 大幅度地增強了 VuePress 的可扩展性。本项目是一個 VuePress 插件，由 VuePress 1.x 的作者创建，其目的是盡可能地降低 VuePress 博客主题的开发成本。
```

File: docs/zh/guide/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+请注意，此插件已深入集成到 [@vuepress/theme-blog](https://github.com/ulivz/vuepress-theme-blog).
```
Comment (newsbielt703) [inline docs/zh/guide/README.md:7]:
```suggestion
请注意，本插件已深入集成到 [@vuepress/theme-blog](https://github.com/ulivz/vuepress-theme-blog).
```

File: docs/zh/guide/README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+为了了解此插件的用法，我们建议你首先了解 VuePress ，并且你最好具有使用 VuePress 的经验。你可以通过以下方式了解 VuePress 1.x：
```
Comment (newsbielt703) [inline docs/zh/guide/README.md:9]:
```suggestion
为了了解本插件的用法，我们建议你先对于 VuePress 有些认识 ，而使用过 VuePress 的话更好。你可以通过以下方式了解 VuePress 1.x：
```

**Comment by Mister-Hope** [comment]
By the way, I made a fix for the navbar dropdown issue in 1.4. As now it's not fixed in the newest version, shall we review the PR and publish a new version to fix i?

**Review by ulivz — APPROVED** [review]
Thanks for the great work!

**Pull request merged**